URBNShareKit.podspec

```diff
@@ -1,28 +1,15 @@
-#
-# Be sure to run `pod lib lint URBNShareKit.podspec' to ensure this is a
-# valid spec and remove all comments before submitting the spec.
-#
-# Any lines starting with a # are optional, but encouraged
-#
-# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
-#
-
 Pod::Spec.new do |s|
   s.name             = "URBNShareKit"
-  s.version          = "0.1.0"
-  s.summary          = "A short description of URBNShareKit."
+  s.version          = "0.1"
+  s.summary          = "URBNShareKit makes using the UIActivityViewController to share a breeze."
   s.description      = <<-DESC
-                       An optional longer description of URBNShareKit
-
-                       * Markdown format.
-                       * Don't worry about the indent, we strip it!
+                       URBNShareKit is a subclass of UIActivityViewController which makes it easier to display & fill the native iOS share sheet with share activities for iOS 7+. Note some activities will only display on iOS 8+ devices.
+		       It also under the scenes will create and handle the UIActivityItemProviders. You just pass the appropriate body text, image, and/or Url.
                        DESC
   s.homepage         = "https://github.com/<GITHUB_USERNAME>/URBNShareKit"
-  # s.screenshots     = "www.example.com/screenshots_1", "www.example.com/screenshots_2"
   s.license          = 'MIT'
-  s.author           = { "Ryan Garchinsky" => "ryang1428@gmail.com" }
-  s.source           = { :git => "https://github.com/<GITHUB_USERNAME>/URBNShareKit.git", :tag => s.version.to_s }
-  # s.social_media_url = 'https://twitter.com/<TWITTER_USERNAME>'
+  s.author           = { "urbn" => "rgarchinsky@urbn.com" }
+  s.source           = { :git => "https://github.com/urbn/URBNShareKit.git", :tag => s.version.to_s }
 
   s.platform     = :ios, '7.0'
```

Comment:
Yes. iOS 7 will not be able to support 3rd party extensions but it will work with the built in extensions